tools/generators/ethereum/command.go.tmpl

```diff
@@ -241,7 +244,7 @@ func initialize{{.Class}}(c *cli.Context) (*contract.{{.Class}}, error) {
         address,
         key,
         client,
-        ethutil.NewNonceManager(key.Address, client),
+        ethutil.NewNonceManager(client, key.Address),
```

Comment:
I think this line will fail compilation. From `nonce_manager.go`:
```
func NewNonceManager(
	account Address,
	transactor ContractTransactor,
) *NonceManager {
```

Reply:
Nope, `ethutil.NewNonceManager` takes the transactor as the first argument. This one has been tested as well.

Reply:
Ah, ok this is twisted. `ethutil.NewNonceManager` takes `EthereumClient + common.Address` while `ethlike.NonceManager` takes `Address + ContractTransactor`. Would it be possible to align the order?

Reply:
You mentioned the code of the `ethlike.NewNonceManager`. I will change its param order to be consistent with the `ethutil.NewNonceManager`.

Reply:
Yes, I was confused by the order. I was looking at `NewNonceManager` in `ethlike`.

Reply:
Done https://github.com/keep-network/keep-common/commit/6949bf58ebd8c74ee3cbe648cc1ce4eeb2cf6c42